rfcs/text/0000-create-file-mach-o.md

```diff
@@ -0,0 +1,150 @@
+# 0000: Create the Mach-O sub-field of the File fieldset
+
+- Stage: **0 (strawperson)**
+- Date: **TBD**
+
+Create the Mach Object (Mach-O) sub-field, of the `file` top-level fieldset. This document metadata can be used for malware research, as well as coding and other application development efforts.
```

Comment:
Perhaps `macho` could be added as a new top-level field set and nested under `file.*`, similar to the `pe.*` fields?

Reply:
Yep, that's the intention.

I've gone back and cleaned up the `file.mach.*` that was throughout the doc/yml file...I made the same mistake with `pe.*`